project/Settings.scala

```diff
@@ -113,12 +113,5 @@ object Settings {
     )
   )
 
-  lazy val newrelicSettings = List(
```

Comment:
leaving it out for now since welder is a service deployed on user's cluster. It's a bit more complicated enabling NR in this case. (we need a NR license key on the vm, which we were talking about potentially giving user root access)